sidebars.js

```diff
@@ -73,18 +73,34 @@ const sidebars = {
     { type: 'autogenerated', dirName: 'integrations/social' },
     {
       type: 'link',
-      label: 'SMS and email connectors',
+      label: 'Email connectors',
       href: '#',
       className: 'sidebar-section',
     },
-    { type: 'autogenerated', dirName: 'integrations/sms-and-email' },
+    { type: 'autogenerated', dirName: 'integrations/email' },
     {
       type: 'link',
-      label: 'SSO connectors',
+      label: 'SMS connectors',
       href: '#',
       className: 'sidebar-section',
     },
-    { type: 'autogenerated', dirName: 'integrations/sso' },
+    { type: 'autogenerated', dirName: 'integrations/sms' },
+    // {
+    //   type: 'link',
+    //   label: 'Enterprise connectors',
+    //   customProps: { additionalLabel: '(SP-initiated SSO)' },
+    //   href: '#',
+    //   className: 'sidebar-section',
+    // },
+    // { type: 'autogenerated', dirName: 'integrations/sso' },
+    // {
+    //   type: 'link',
+    //   label: 'Third-party apps',
+    //   customProps: { additionalLabel: '(Logto as Idp)' },
+    //   href: '#',
+    //   className: 'sidebar-section',
+    // },
+    // { type: 'autogenerated', dirName: 'integrations/third-party' },
```

Comment:
why is this commented?

Reply:
The SSO and 3rd-party app guides are available in the follow-up PR #668.